src/plugins/template/src/plugin.cpp

```diff
@@ -258,7 +261,10 @@ ov::Any ov::template_plugin::Plugin::get_property(const std::string& name, const
                                                     ov::hint::execution_mode,
                                                     ov::num_streams,
                                                     ov::template_plugin::disable_transformations,
-                                                    ov::log::level};
+                                                    ov::log::level,
+                                                    ov::enable_mmap,
+                                                    ov::compilation_num_threads,
+                                                    ov::inference_num_threads};
```

Comment:
can we enable some tests to check newly added properties?
@sbalandi  could you please point where to enable such tests?

Reply:
I think ov_api_conformance_tests cover those properties. Should I add additional ones? @sbalandi Let me know if you have any pointers.

https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/plugin/shared/src/behavior/ov_plugin/properties_tests.cpp#L304